src/__experimental__/components/help/help.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import 'jest-styled-components';
+import Help from './help.component';
+
+function render(props) {
+  return TestRenderer.create(<Help { ...props } />);
+}
+
+describe('Help', () => {
+  it('renders as expected', () => {
+    expect(render()).toMatchSnapshot();
+  });
+
+  it('sets appropriate className when passed an href', () => {
```

Comment:
You could probably test this without a snapshot.

Something like `expect(wrapper.find('class')).toExist()`.